<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AlliesDialog</class>
 <widget class="QDialog" name="AlliesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>740</width>
    <height>244</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Allies</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>20</y>
     <width>491</width>
     <height>191</height>
    </rect>
   </property>
   <property name="title">
    <string>Teams</string>
   </property>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>111</width>
      <height>161</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page1">
     <widget class="QListWidget" name="teamAList">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>91</width>
        <height>131</height>
       </rect>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::CustomContextMenu</enum>
      </property>
      <property name="whatsThis">
       <string>Delete player from this team by right clicking it and then click 'Delete'.</string>
      </property>
      <property name="dragEnabled">
       <bool>true</bool>
      </property>
      <property name="dragDropMode">
       <enum>QAbstractItemView::DragDrop</enum>
      </property>
      <property name="defaultDropAction">
       <enum>Qt::MoveAction</enum>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="AButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>A</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page2">
     <widget class="QListWidget" name="teamEList">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>91</width>
        <height>131</height>
       </rect>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::CustomContextMenu</enum>
      </property>
      <property name="whatsThis">
       <string>Delete player from this team by right clicking it and then click 'Delete'.</string>
      </property>
      <property name="dragEnabled">
       <bool>true</bool>
      </property>
      <property name="dragDropMode">
       <enum>QAbstractItemView::DragDrop</enum>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="EButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>E</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget_2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>111</width>
      <height>161</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page1_2">
     <widget class="QListWidget" name="teamBList">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>91</width>
        <height>131</height>
       </rect>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::CustomContextMenu</enum>
      </property>
      <property name="whatsThis">
       <string>Delete player from this team by right clicking it and then click 'Delete'.</string>
      </property>
      <property name="dragEnabled">
       <bool>true</bool>
      </property>
      <property name="dragDropMode">
       <enum>QAbstractItemView::DragDrop</enum>
      </property>
      <property name="defaultDropAction">
       <enum>Qt::MoveAction</enum>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="BButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>B</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page2_2">
     <widget class="QListWidget" name="teamFList">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>91</width>
        <height>131</height>
       </rect>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::CustomContextMenu</enum>
      </property>
      <property name="whatsThis">
       <string>Delete player from this team by right clicking it and then click 'Delete'.</string>
      </property>
      <property name="dragEnabled">
       <bool>true</bool>
      </property>
      <property name="dragDropMode">
       <enum>QAbstractItemView::DragDrop</enum>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="FButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>F</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget_3">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>20</y>
      <width>111</width>
      <height>161</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page1_3">
     <widget class="QListWidget" name="teamCList">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>91</width>
        <height>131</height>
       </rect>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::CustomContextMenu</enum>
      </property>
      <property name="whatsThis">
       <string>Delete player from this team by right clicking it and then click 'Delete'.</string>
      </property>
      <property name="dragEnabled">
       <bool>true</bool>
      </property>
      <property name="dragDropMode">
       <enum>QAbstractItemView::DragDrop</enum>
      </property>
      <property name="defaultDropAction">
       <enum>Qt::MoveAction</enum>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="CButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>C</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page2_3">
     <widget class="QListWidget" name="teamGList">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>91</width>
        <height>131</height>
       </rect>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::CustomContextMenu</enum>
      </property>
      <property name="whatsThis">
       <string>Delete player from this team by right clicking it and then click 'Delete'.</string>
      </property>
      <property name="dragEnabled">
       <bool>true</bool>
      </property>
      <property name="dragDropMode">
       <enum>QAbstractItemView::DragDrop</enum>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="GButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>G</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget_4">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>20</y>
      <width>111</width>
      <height>161</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page1_4">
     <widget class="QListWidget" name="teamDList">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>91</width>
        <height>131</height>
       </rect>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::CustomContextMenu</enum>
      </property>
      <property name="whatsThis">
       <string>Delete player from this team by right clicking it and then click 'Delete'.</string>
      </property>
      <property name="dragEnabled">
       <bool>true</bool>
      </property>
      <property name="dragDropMode">
       <enum>QAbstractItemView::DragDrop</enum>
      </property>
      <property name="defaultDropAction">
       <enum>Qt::MoveAction</enum>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="DButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>D</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page2_4">
     <widget class="QListWidget" name="teamHList">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>91</width>
        <height>131</height>
       </rect>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::CustomContextMenu</enum>
      </property>
      <property name="whatsThis">
       <string>Delete player from this team by right clicking it and then click 'Delete'.</string>
      </property>
      <property name="dragEnabled">
       <bool>true</bool>
      </property>
      <property name="dragDropMode">
       <enum>QAbstractItemView::DragDrop</enum>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="HButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>H</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>221</width>
     <height>191</height>
    </rect>
   </property>
   <property name="title">
    <string>Players</string>
   </property>
   <widget class="QListWidget" name="nonTeamList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>201</width>
      <height>161</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="whatsThis">
     <string>Drag and drop players from this list to the desired team.</string>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="dragEnabled">
     <bool>true</bool>
    </property>
    <property name="dragDropOverwriteMode">
     <bool>false</bool>
    </property>
    <property name="dragDropMode">
     <enum>QAbstractItemView::DragOnly</enum>
    </property>
    <property name="defaultDropAction">
     <enum>Qt::CopyAction</enum>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="viewMode">
     <enum>QListView::ListMode</enum>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>495</x>
     <y>215</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>655</x>
     <y>215</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clearButton">
   <property name="geometry">
    <rect>
     <x>575</x>
     <y>215</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
